### DISCUSSION TITLE
Equal Opportunities

### DISCUSSION THREAD

**Discussion by Alaindrea** [sunflower-land/sunflower-land#2935] [Ideas]
While I am pretty sure I'm going to be shouted down by the elite players and whales who own exactly the kind of rare items I'm talking about, and that those are the players who are most likely to be here in github, I'm going to try this anyway. 

I would like it if once or twice a year a rare item with a good boost (like CG, GA, Wings, etc) were made **equally** available to everyone. 

Let me back up and start with rationale behind that statement. Early adopters have had a continuous advantage that translates into them having some of the rarest items in the game. And deservedly so. They believed in the game when it wasn't nearly as fun or exciting as it is today, and stayed with it even when things changed. As such, they've been rewarded, as they should have been. 

------AND-----

These items give them an advantage in every other competition for the next set of rare items- the players most likely to get rare items are players who already _have_ rare items. They are also virtually guaranteed to get the top items in a race (cabbage girl, turtle, etc) and to earn the most entries on tasks for tickets either in a drawing (like Hoot) or Auction (feather auctions, more resources generated to sell for SFL auctions). The only way to "compete" with these players is to buy your way in by buying resources to get onto the leaderboard. Or to buy the item from one of those players who is willing to sell theirs, often because they have a different item with a similar boost that they prefer or a set of items that equate to them preferring to specialize in a certain resource.  

As a beta player with a coop I fully admit that I have also benefited from this, I was able to win a guaranteed mint of buds for eggs and also got whitelist mint for because I had a coop. _1X_ since beta I have actually had an advantage that allowed me to successfully compete for an item. And I'm grateful for that. And I'm also not saying this should _never_ happen, or that players with rare items should _never_ have an advantage.

-----BUT----

There have been far, far more times when I have worked as hard as I could and literally not had a chance. For example, I would have killed for a Hoot, before I even knew what the boost was, because I love owls. I worked as hard as I could and got over 30 tickets one week. When the drawing was held I won 1000 SFL that I immediately used to try to get myself on the leaderboard in one of the top 5 places. In the last half day I was knocked off completely- even with the 1000 SFL I just couldn't compete.  That's just one example.  This was not an isolated thing. It has happened repeatedly and I have been active since Beta. If that's what it is usually like for me,  what does that feel like to an even newer player who doesn't even have the advantages I do? I'm going to take a guess and say completely hopeless for many- especially because more than one person I have referred to the game told me as much, and that that wasn't fun. 

It is just as important to retain players as it is to attract new players. If we want a game to be fun for everyone (elite, new, and mid tier players) then the playing field needs to be level **_sometimes_**. Not _every time_, but also not _never_. As I said, the same people keep "winning" because their previous "wins" give them advantages in the next contest. It is like if you were playing soccer, and the losing team wasn't allowed any water at the next match. 

That's the problem as I see it. At some point, players who don't have the advantages need an equal shot to get an elite (i.e. rare item with a good boost) item without a significant cash outlay to other players. The only solution to this is to release an elite item periodically that basically everyone has the same chance to win/earn. This allows more people to move up either by obtaining an elite item that they can use themselves in the future, or that they can sell to get other items that they think will best help them advance. 

**I think there are two possible solutions for how this could be done.** I think one method is superior, but I'll offer both for consideration. I hope others will offer still more.

The first is a **skill based contest**, something like the maze where basically each player gets a certain number or unlimited number of tries at a flat rate that is affordable to even new players (for example: 100 sunflowers, or 1 sfl) or for free. The winners on the **leaderboard** each week or the top player each day would get the item.   Other skill/game options could be an asynchronous two player game (like a rock paper scissors) **bracketed tournament**.

It would NOT be fair if certain items gave some players an advantage or allowed them to impede other players (flying over obstacles or acting as a bad guy like crumple crown).  This is not the method I prefer, because I'm concerned that players who can only play on one device (phone or laptop), players who don't have access to a fast, reliable internet connection, and any player who has a relevant physical disability may be significantly disadvantaged. I would absolutely say PVP is NOT a vibe that welcoming to many players. Two player games that aren't asynchronous have many of the same issues as skill/timed games against the computer for people who don't have the same devices or access. 

The method I would prefer is an **equal drawing.** For example, all players with a gold pass who are of sufficient level to access Bumpkin Plaza would be eligible to enter. They could enter by once a day talking to an NPC in Bumpkin Plaza. Every player who checked into that NPC would get 1 entry and there would be no way to transfer the entries and no way to get more. Essentially all active players would have an equal chance, whether they were level 5 or level 50, and regardless of what other game items they had or how many resources they hoarded or purchased.  I realize this also has potential drawbacks, such as people creating multiple accounts to try to get more chances and that checking in once a day may not be as enjoyable as competing or grinding to achieve a goal for some. 

(I suppose it would also be possible to group or handicap players by level, but that has been discussed before and it continues to be problematic for multiple reasons)

Feel free to share your ideas, alternate solutions, and/or feedback

**Comment by brian-man** [comment]
Thanks for putting yourself and your effort out there to raise the discussion.

Some initial thoughts:
1. "once or twice a year a rare item with a good boost (like CG, GA, Wings, etc) were made equally available to everyone." -- I'm pretty sure the team never intends to increase the supply of an existing item.  What they're more likely to do is introduce a "seasonal sibling" that is slightly less powerful.  For example, for Green Amulet that might be Lime Amulet that has a slightly lower proc rate.
2. "items give them an advantage in every other competition for the next set of rare items" - I've made this point a few times to the team but so far they've suggested they like this.  My preference is that items give advantage on efficiency and yield, but not direct advantage on acquisition of new items (via ticket collection rate, for example).  I think the only real solution is to make all "races" time-limited rather than supply-limited.  If 1000 players put in ridiculous effort to achieve something, the supply should match that "effort demand" rather than be artificially limited to some small number like 50.
3. "times when I have worked as hard as I could and literally not had a chance" -- Agreed.  This aspect of many of the races is bad, especially if the game wants to keep growing by adding and retaining players.
4. "It would NOT be fair if certain items gave some players an advantage or allowed them to impede other players (flying over obstacles or acting as a bad guy like crumple crown)." -- I completely agree the new "PVP" mechanic planned for the maze is terrible in this regard.  There's a reason why I've always hated games that built PVP systems that attracts griefers / bullies.  It's unfortunate Adam chose to move in this direction.
5. "The method I would prefer is an equal drawing." -- Personally, this model is completely unappealing to me. It basically says "we don't know how to build a compelling game, or reward effort or skill, so we'll just make it RNG".  The more the game moves in this direction the more it loses my interest and I start planning my exit.

Cheers!

**Reply by Alaindrea** [reply]
Just want to clarify that I wasn't suggesting re-issuing any existing item, but rather a new one(s) be distributed in a different way.  Thanks for the well thought response :)

**Comment by TheDcol** [comment]
The GA mint was something achievable for a mid level player to get, it did come down to gas/blockchain luck in the end which is 1 of the many reasons the team switched to a seasonal event/race to acquire items

Good points brought up!
